esp/esp/settings.py

```diff
@@ -51,6 +51,7 @@
 if len(DATABASES['default']['PASSWORD']) == 0:
     try:
         DATABASES['default']['PASSWORD'] = DATABASE_PASSWORD
+        DATABASES['default']['PASSWORD'] = DATABASE_PASSWORD
```

Comment:
Why was a copy of this line added? I can't think of any reason other than it being accidental.